src/Databases/DatabaseReplicatedWorker.cpp

```diff
@@ -249,6 +250,8 @@ String DatabaseReplicatedDDLWorker::enqueueQueryImpl(const ZooKeeperPtr & zookee
         }
         else if (code != Coordination::Error::ZNODEEXISTS)
             zkutil::KeeperMultiException::check(code, ops, res);
+
+        sleepForMilliseconds(50);
```

Comment:
In description `sleep for 10ms` here 50. 
Maybe we shall add a server setting for it with default 50?